volumina/volumeEditor.py

```diff
@@ -139,7 +139,7 @@ def lastImageViewFocus(self, axis):
         self._lastImageViewFocus = axis
         self.newImageView2DFocus.emit()
 
-    def __init__( self, layerStackModel, parent, labelsink=None, crosshair=True, syncAlongAxes=(0,1)):
+    def __init__( self, layerStackModel, parent, labelsink=None, crosshair=True, add_3d_widget=False, syncAlongAxes=(0,1)):
```

Comment:
Naming the parameter `add_3d_widget` is a bit unfortunate; a 3D view is added in any case, it's just not visible, if I understand the code correctly. Maybe sticking to `is_3d_visible` or even more explicit like `3d_widget_visible` would be better

Reply:
totally agree!

Reply:
✅